.github/workflows/build.yml

```
@@ -9,7 +9,9 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    uses: [actions/checkout@v3, ./.github/actions/prepare]
     steps:
-      - uses: actions/checkout@v3
-      - uses: ./.github/actions/prepare
-      - run: pnpm build
+      - id: test
+        run: |
+          pnpm build
+          node ./src/index.js
```

There's no need to combine these commands into a single `run` with an `id`. Just have two `run`s with one command each. Then the title for each `run` will be the command!

```suggestion
      - id: test
        run: pnpm build
        run: node ./src/index.js
```